tests/integrations/raftstore/test_witness.rs

```diff
@@ -0,0 +1,425 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::{iter::FromIterator, sync::Arc, time::Duration};
+
+use futures::executor::block_on;
+use kvproto::metapb;
+use pd_client::PdClient;
+use test_raftstore::*;
+use tikv_util::store::find_peer;
+
+fn become_witness(cluster: &Cluster<ServerCluster>, region_id: u64, peer: &mut metapb::Peer) {
```

Comment:
If the implementation is changed from add/remove to the new command, there will be no case covering add/remove.

Reply:
check out  `add a new witness peer` in `test_witness_conf_change`

Reply:
Why conf change depends on switch witness? 🤣